### PR TITLE
Update readme with steps to install the collection from AH

### PR DESCRIPTION
**Pull request by GomathiselviS**
Reference Jira Ticket : https://issues.redhat.com/browse/ACA-1017

The validated content collections are now available in Automation Hub. The installation steps are updated in the README.

### CODE REVIEW

**Comment by GomathiselviS** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -33,12 +33,23 @@ The [amazon.aws](https://github.com/ansible-collections/amazon.aws) and [communi
 
 
 ### Installation
-Clone the collection repository.
 
-```shell
-  mkdir -p ~/.ansible/collections/ansible_collections/cloud/aws_troubleshooting
-  cd ~/.ansible/collections/ansible_collections/cloud/aws_troubleshooting
-  git clone https://github.com/redhat-cop/cloud.aws_troubleshooting .
```
Comment (alinabuzachis) [inline README.md:41]: What if the user does not use the collection from AH?

Comment (GomathiselviS) [reply README.md:41]: > What if the user does not use the collection from AH?

Hi @alinabuzachis , Based on the Slack conversations in #ansible-partners,  it seems like the consensus is to promote the use of Automation Hub. This seems to be the approach for other collections too.

**Comment by GomathiselviS** [comment]
recheck

File: README.md (hunk 1 above)
Comment (jillr) [reply README.md:41]: The Validated Content program does put a focus on platform experience, so I agree we should show AH instructions in the readme.

**Review by hakbailey — APPROVED** [review]

**Review by alinabuzachis — APPROVED** [review]

**Review by jillr — APPROVED** [review]

**Pull request merged**